test/e2e/node/ttlafterfinished.go

```diff
@@ -68,12 +68,13 @@ func testFinishedJob(f *framework.Framework) {
 	ns := f.Namespace.Name
 	c := f.ClientSet
 
+	isSuspend := false
 	parallelism := int32(1)
 	completions := int32(1)
 	backoffLimit := int32(2)
 	ttl := int32(10)
 
-	job := e2ejob.NewTestJob("randomlySucceedOrFail", "rand-non-local", v1.RestartPolicyNever, parallelism, completions, nil, backoffLimit)
+	job := e2ejob.NewTestJob("randomlySucceedOrFail", "rand-non-local", v1.RestartPolicyNever, isSuspend, parallelism, completions, nil, backoffLimit)
```

Comment:
I don't think an ever growing `NewTestJob` function is ideal. Specially since most tests don't care about the suspend flag.
Instead, just override the extra value where necessary, like it's done for ttl down below